spec/classes/cron_spec.rb

```diff
@@ -144,4 +144,93 @@
       )
     }
   end
+
+  context 'manage_crontab => true' do
+    let :facts do
+      {
+        operatingsystem: 'Unsupported'
```

Comment:
could you instead convert this to rspec_puppet_facts?

url: https://github.com/mcanevet/rspec-puppet-facts
example: https://github.com/voxpupuli/puppet-zabbix/blob/master/spec/classes/database_mysql_spec.rb#L12

Reply:
I only updated the file I added tests to , others probably more appropriate for different PR